styles.css

```diff
@@ -293,28 +282,41 @@ article{
 	font-weight:700;
 }
 
+.clearfix {
+	display:block;
+	height:1%;
+}
+
+.clearfix:after {
+	content:".";
+	/*display:block;*/
```

Comment:
take unnecessary comments out of live code

Reply:
resolved

Reply:
revolved